Cooper.DAO/GameDAO.cs

```diff
@@ -34,13 +34,22 @@ public GameDAO(IConfigProvider configProvider)
 
         #region Get methods
 
-        public GameDb Get(object id)
+        public GameDb Get(object obj)
         {
+            string name = obj.ToString();

```

Comment:
?
if obj is some class like array - you will have name="String[]"
Why can't we make a condition `if (obj is string name) { //find by name} else if (obj is long id) {...} else {some error}`